### ISSUE TITLE
Won't work with Markdown v3.x.x

### ISSUE THREAD

**Issue by brunobord** [ionelmc/python-darkslide#10]
Steps to reproduce:

```console
$ pip install darkslide
$ darkslide index.md
Adding   'index.md' (markdown)
Error: markdown() takes exactly 1 argument (2 given)
```

Here is the output of ``pip freeze -l``

```
pip freeze -l
darkslide==4.0.1
docutils==0.14
Jinja2==2.10
Markdown==3.0.1
MarkupSafe==1.0
Pygments==2.2.0
qrcode==6.0
six==1.11.0
```

If you install a previous version of Markdown instead (e.g. ``pip install Markdown==2.6.11``, it solves the problem.
It comes apparently from this line: https://github.com/ionelmc/python-darkslide/blob/master/src/darkslide/parser.py#L61 ; it looks like the markdown function accepts kwargs, but no more implicit args. So I guess that a quick fix would be:

```python
return markdown.markdown(text, extensions=self.md_extensions)
```

reference: https://python-markdown.github.io/reference/#extensions

**Comment by GaelVaroquaux** [comment]
Is there a plan to address this issue? Is this an error on the user side?

**Comment by GaelVaroquaux** [comment]
> I forgot to close this. It was fixed in 926abac and released a v5.0

OK, thank you!
